### PR TITLE
Add install python role

### PR DESCRIPTION
**Pull request by Yikun**
Related: https://github.com/theopenlab/openlab-zuul-jobs/pull/618
Related: https://github.com/theopenlab/openlab-zuul-jobs/pull/620

### CODE REVIEW

File: roles/config-python/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+python_major_version: '3'
```
Comment (wangxiyuan) [inline roles/config-python/defaults/main.yml]: this is unnecessary.
try:
```
python_version= {{ python_version }}
echo ${python_version:0:1}
```

Comment (Yikun) [reply roles/config-python/defaults/main.yml]: done. `major_version=`echo {{ python_version }} | cut -d '.' -f1``

File: roles/config-python/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# We can install specify version gcc/g++ in task, like this:
```
Comment (liusheng) [inline roles/config-python/tasks/main.yml]: it is better to check if python3 has already installed and skip 'Install python' task if yes.

File: roles/config-python3/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    set -ex
+    # 3.5 ==> 3
+    major_version=`echo {{ python_version }} | cut -d '.' -f1`
+    # Install python{{ python_version }}
+    add-apt-repository ppa:deadsnakes/ppa -y
+    apt-get update
+    apt-get install python{{ python_version }} python$major_version-dev libssl-dev -y
+    update-alternatives --install $(which python$major_version) python$major_version $(which python{{ python_version }}) 1
+  when:
+    - curr_python_version.stdout != python_version
```
Comment (liusheng) [inline roles/config-python3/tasks/main.yml]: not sure if  version number `lower` better than `!=`

Comment (Yikun) [reply roles/config-python3/tasks/main.yml]: As F2F discussion, It's better to match the specific version.

**Pull request merged**